app/views/resources/show.html.erb

```diff
@@ -21,6 +21,7 @@
                          edit_path: backend.url_for([:edit, @media_content]) }
                       ]
                       end
+  users = resource_users(@media_content)
```

Comment:
This might be better as a method of the MediaContent model maybe, as in: `@media_content.users` (in which case you won't even need a local var in the view)